### PR TITLE
Add independent library targets for newly added .h files for microben…

### PR DESCRIPTION
**Pull request by kumaralokgithub**
…chmarks. Also fix includes in fullstack_fixtures.h

### CODE REVIEW

File: test/cpp/microbenchmarks/BUILD [hunk 1]
```diff
@@ -73,12 +73,35 @@ grpc_cc_binary(
     deps = [":helpers"],
 )
 
+grpc_cc_library(
+    name = "fullstack_streaming_ping_pong_h",
+    testonly = 1,
+    srcs = [
+        "fullstack_streaming_ping_pong.h",
+    ],
```
Comment (vjpai) [inline test/cpp/microbenchmarks/BUILD]: If it's a header-only library, I don't think `srcs = ` is necessary, is it? We don't use it, for example, in some targets at the top level like `gpr_codegen`

File: test/cpp/microbenchmarks/fullstack_fixtures.h [hunk 2]
```diff
@@ -41,6 +41,7 @@ extern "C" {
 #include "test/core/util/port.h"
 }
 
+#include "src/cpp/client/create_channel_internal.h"
```
Comment (vjpai) [inline test/cpp/microbenchmarks/fullstack_fixtures.h]: If you're adding it here, please remove it from the other places where it is used (each of the fullstack_*.h files that includes this fixtures header)

File: test/cpp/microbenchmarks/BUILD (hunk 1 above)
Comment (kumaralokgithub) [reply test/cpp/microbenchmarks/BUILD]: Done

File: test/cpp/microbenchmarks/fullstack_fixtures.h (hunk 2 above)
Comment (kumaralokgithub) [reply test/cpp/microbenchmarks/fullstack_fixtures.h]: Done

**Comment by markdroth** [comment]
Looks like you've got a bad merge here somehow.  This PR seems to include changes from several other recent PRs.

**Comment by kumaralokgithub** [comment]
The merge is fixed now.
dgquintas, ctiller, markdroth: Please, ignore the review request. The tool is not allowing me to remove you from reviewers list.

**Pull request merged**